appeals/web/src/server/lib/mappers/appellantCase.mapper.js

```diff
@@ -331,48 +331,19 @@ export function initialiseAndMapData(appellantCaseData, appealDetails, currentRo
 	mappedData.changedDevelopmentDescriptionDocument = {
 		id: 'changed-development-description.document',
 		display: {
-			summaryListItem: {
-				key: {
-					text: 'Agreement to change description evidence'
-				},
-				value: displayPageFormatter.formatDocumentValues(
+			summaryListItem: displayPageFormatter.formatFolderSummaryListItem(
+				'Agreement to change description evidence',
+				appellantCaseData.appealId,
+				appellantCaseData.documents.changedDescription,
+				permissionNames.updateCase,
+				session,
+				mapDocumentManageUrl(
 					appellantCaseData.appealId,
-					isFolderInfo(appellantCaseData.documents.changedDescription)
-						? appellantCaseData.documents.changedDescription.documents
-						: []
+					appellantCaseData.documents.changedDescription?.folderId
 				),
-				actions: {
-					items: [
-						...((
-							(isFolderInfo(appellantCaseData.documents.changedDescription) &&
-								appellantCaseData.documents.changedDescription.documents) ||
-							[]
-						).length
-							? [
-									mapActionComponent(permissionNames.updateCase, session, {
-										text: 'Manage',
-										visuallyHiddenText: 'Agreement to change description evidence',
-										href: mapDocumentManageUrl(
-											appellantCaseData.appealId,
-											isFolderInfo(appellantCaseData.documents.changedDescription)
-												? appellantCaseData.documents.changedDescription.folderId
-												: undefined
-										)
-									})
-							  ]
-							: []),
-						mapActionComponent(permissionNames.updateCase, session, {
-							text: 'Add',
-							visuallyHiddenText: 'Agreement to change description evidence',
-							href: displayPageFormatter.formatDocumentActionLink(
-								appellantCaseData.appealId,
-								appellantCaseData.documents.changedDescription,
-								documentUploadUrlTemplate
-							)
-						})
-					]
-				}
-			}
+				documentUploadUrlTemplate,
+				'add-agreement-to-change-description-evidence'
```

Comment:
first 4 characters should be removed ("add-") as it gets prefixed in "formatFolderSummaryListItem"